cubic-server/ai/wandering.cpp

```diff
@@ -0,0 +1,57 @@
+#include "wandering.hpp"
+#include "Dimension.hpp"
+#include "entities/Entity.hpp"
+#include "logging/logging.hpp"
+#include "math/Vector3.hpp"
+#include "utility/PseudoRandomGenerator.hpp"
+#include "world_storage/Level.hpp"
+
+namespace ai {
+Wandering::Wandering(Entity &entity):
+    AI(entity)
+{
+}
+
+Wandering::~Wandering() { }
+
+bool Wandering::see() { return true; }
+
+void Wandering::think()
+{
+    Vector3<double> actualPos = this->_entity.getPosition();
+
+    int x = utility::PseudoRandomGenerator::getInstance()->generateNumber(-9, 9);
+    int z = utility::PseudoRandomGenerator::getInstance()->generateNumber(-9, 9);
+
+    Vector3<double> finalPos = {actualPos.x + x, actualPos.y, actualPos.z + z};
```

Comment:
This should check that the place is usable as an end block, so that the chicken doesn't end up inside blocks or up in the air at the end of its walk.

Reply:
it's done just after ^^